WebDevStudios/constant-contact-woocommerce
# CC-141: Add coding standards

Ticket: [CC-141](https://webdevstudios.atlassian.net/browse/CC-141)

### Description ###

Adds WP, WDS coding standards and PHPCS annotated ruleset file.

### Steps to test ###

Run `npm install && composer install` locally then `composer run lint` and confirm a) linting runs as expected and b) no errors display.

Note: There were some weird errors on my local while testing, but a fresh repo clone showed no remaining errors.
